yew-macro/src/html_tree/html_tag/tag_attributes.rs

```diff
@@ -54,49 +54,108 @@ lazy_static! {
     static ref LISTENER_SET: HashSet<&'static str> = {
         HashSet::from_iter(
             vec![
+                // Living Standard
+                // From: https://html.spec.whatwg.org/multipage/webappapis.html#globaleventhandlers
+                "onabort",
+                "onauxclick",
+                "onblur",
+                "oncancel",
+                "oncanplay",
+                "oncanplaythrough",
+                "onchange",
                 "onclick",
-                "ondoubleclick",
-                "onkeypress",
+                "onclose",
+                "oncontextmenu",
+                "oncuechange",
+                "ondoubleclick", // renamed from "ondblclick" for compatibility
```

Comment:
Let's rename to `ondblclick`, we can call this out in the release notes. Yew is still young enough that we don't need to carry forward tech debt like this

Reply:
That's a great idea. Will adjust when I get the time

Reply:
Great, take your time 👍